test/test-utils.js

```diff
@@ -565,3 +575,82 @@ export const createVoucher =
       throw error;
     }
   };
+
+export const authorizeAccount =
+  (chain) => async (voucherAddress, accountAddress) => {
+    const voucherContract = new Contract(
+      voucherAddress,
+      [
+        {
+          inputs: [],
+          name: 'owner',
+          outputs: [
+            {
+              internalType: 'address',
+              name: '',
+              type: 'address',
+            },
+          ],
+          stateMutability: 'view',
+          type: 'function',
+        },
+        {
+          inputs: [
+            {
+              internalType: 'address',
+              name: 'account',
+              type: 'address',
+            },
+          ],
+          name: 'authorizeAccount',
+          outputs: [],
+          stateMutability: 'nonpayable',
+          type: 'function',
+        },
+      ],
+      chain.provider,
+    );
+    const owner = await voucherContract.owner();
+    await impersonate(owner);
+    const authorizeAccountTxHash = await voucherContract
+      .connect(new JsonRpcSigner(chain.provider, owner))
+      .authorizeAccount(accountAddress, { gasPrice: 0 });
+    await authorizeAccountTxHash.wait();
+    await stopImpersonate(owner);
+    console.log(
+      `account ${accountAddress} added to authorizedAccounts for voucher ${voucherAddress} (tx: ${authorizeAccountTxHash.hash})`,
+    );
+  };
+
+export const addEligibleAsset =
+  (chain) => async (assetAddress, voucherTypeId) => {
+    const voucherHubContract = new Contract(VOUCHER_HUB_ADDRESS, [
+      {
+        inputs: [
+          {
+            internalType: 'uint256',
+            name: 'voucherTypeId',
+            type: 'uint256',
+          },
+          {
+            internalType: 'address',
+            name: 'asset',
+            type: 'address',
+          },
+        ],
+        name: 'addEligibleAsset',
+        outputs: [],
+        stateMutability: 'nonpayable',
+        type: 'function',
+      },
+    ]);
+
+    const signer = chain.voucherManagerWallet.connect(chain.provider);
```

Comment:
possible front run in using a unique wallet in parallelized tests (nonce collision)
this will also occur in `createVoucherType` and `createVoucher`

example of nonce collision error observed when running tests:
```
  ● voucher test utils › createVoucherType should create a voucherType and return the id

    replacement fee too low (transaction="0xf8e780808301895b94959922be3caee4b8cd9a407cc3ac1c251c2007b180b88425dd8f830000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000002a00000000000000000000000000000000000000000000000000000000000000117465737420766f75636865722074797065000000000000000000000000000000820130a03b30a9fac909a5d574cea67e4f50c90cfda174d59f138ee99d0d1393f83a4f82a0240f2c2ffb80b46faf3cc172c05ea205ef59b0b1f6ef7aee79e40d8ed7d9a531", info={ "error": { "code": -32003, "message": "replacement transaction underpriced" } }, code=REPLACEMENT_UNDERPRICED, version=6.8.0)

      at makeError (node_modules/ethers/src.ts/utils/errors.ts:694:21)
      at JsonRpcProvider.getRpcError (node_modules/ethers/src.ts/providers/provider-jsonrpc.ts:953:24)
      at node_modules/ethers/src.ts/providers/provider-jsonrpc.ts:526:45
```


possible remediation with retries (see `adminCreateCategory` implementation)